gradle/libraries.gradle

```diff
@@ -59,6 +59,7 @@ ext {
 
             // Byte Buddy
             byteBuddy:      "net.bytebuddy:byte-buddy:${byteBuddyVersion}",
+            byteBuddy_agent: "net.bytebuddy:byte-buddy-agent:${byteBuddyVersion}",
```

Comment:
AFAICS, you don't use that one.

Reply:
I don't but it's good to have it defined as a memo - it's pulled in transitively from Mockito.

Reply:
Well, my issue with it is that we will suppose it's what sets the version but it's not at all.

Feel free to push if you think I worry for nothing but I don't like having things misleading the developer. A comment saying that the version of byte-buddy-agent is enforced below would have my preference.

Again, feel free to ignore me on this one :).

Reply:
np, I'll remove it since you think that's clearer.